apps/src/code-studio/pd/teacher_application/applicationConstants.js

```diff
@@ -22,7 +22,7 @@ const group1CspWorkshops = {
   'Nextech': 'July 18 - 22, 2017',
   'The Council of Educational Administrative and Supervisory Organizations of Maryland (CEASOM)': 'August 7 - 11, 2017',
   'Southern Nevada Regional Professional Development Program': 'June 12 - 16, 2017',
-  'Code/Interactive': 'July 10 - 14, 2017',
+  'Code/Interactive': 'August 14 - 18, 2017',
```

Comment:
So districts will be going to either the July 10 or August 14 workshops. The ones in the TSV are specifically going to July 10, all the others are going to August 14